packages/core/resolver/on-command.resolver.ts

```diff
@@ -148,8 +148,8 @@ export class OnCommandResolver implements MethodResolver {
   }
 
   private getCommandName(messageContent: string, prefixLength: number): string {
-    const messageParts = messageContent.split(' ')[0];
-    return messageContent.slice(prefixLength, messageParts.length);
+    const messageWithoutPrefix = messageContent.slice(prefixLength);
+    return messageWithoutPrefix.split(' ').shift() as string;
```

Comment:
Remove please string cast because it is overkill

Reply:
Done! :smile: